docs/cache/develop/integrations/amazon-eventbridge.md

```diff
@@ -146,3 +146,10 @@ When that code runs, it will publish an event to the default event bus, the *Cac
 This integration will continue to grow as our HTTP API gains more and more features. You can always come back here, [deploy the stack to your AWS account](https://console.aws.amazon.com/cloudformation/home#/stacks/create/review?stackName=momento-api&templateURL=https://momento-developers.s3.amazonaws.com/api-destinations.yaml) and get the updated version automatically. 
 
 If you have issues deploying the stack to your account or would like support for another region, feel free to [reach out on Discord](https://discord.com/invite/3HkAKjUZGq) and the team will be happy to help you.
+
+## Explore More
```

Comment:
i like the general idea of what you're doing here but I wonder if we should feature it more prominently?

What if, after the first paragraph of this page, you put one of those "light bulb" boxes (I think there are some scattered around other pages that you could look at as examples), that said something like this:

> On this page you'll learn some general-purpose patterns for connecting EventBridge to Momento. For a more concrete example, check out our [DynamoDB-Momento EventBridge Demo](link)! This project shows how to create a write-through cache in Momento using DynamoDB Streams.

You can include a bit more of your text from below if you like!

I would probably do the same thing on the topics eventbridge page in these docs.

And then the last thing I'm waffling on is whether we should make those links go directly to the github repo, or if we need a full page in the docs site that goes into more detail about your project. Probably github repo link is fine.

Reply:
one other thing on this PR - let's add a link to this from the nodejs overview pages (cache and topics) please!